devops/auto_node/scripts/run.sh

```diff
@@ -0,0 +1,129 @@
+#!/bin/bash
+set -e
+
+unset val_acc_private_key bls_private_key network duration name beacon_endpoint active yes clear generate_shard
+val_acc_private_key=$1
+bls_private_key=''
+network=staking
+beacon_endpoint="https://api.s0.os.hmny.io/"
+active=false
+yes=false
+clear=false
+name=''
+duration=0
+generate_shard=-1
+
+for i in "${@}"
+do
+case $i in
+    -s=*)
+       generate_shard="${i#*=}"
+       shift ;;
+    -K=*)
+       bls_private_key="${i#*=}"
+       shift ;;
+    -N=*)
+       network="${i#*=}"
+       shift ;;
+    -D=*)
+       duration="${i#*=}"
+       shift ;;
+    -n=*)
+       name="${i#*=}"
+       shift ;;
+    -e=*)
+       beacon_endpoint="${i#*=}"
+       shift ;;
+    -a)
+      active=true
+      shift ;;
+    -y)
+      yes=true
+      shift ;;
+    -c)
+      clear=true
+      shift ;;
+    -h | --help)
+      echo '
+      == Main node execution help message ==
+
+      Ex:
+        ./run.sh 0336e9be71c31d71d086d9f0887d13cb6701bc45d11b70bb7c14200c9feebe22 \
```

Comment:
I assume this key is a just a random key, right?

Reply:
Yep just a random key I generated just to test this.